### ISSUE TITLE
Nice job!

### ISSUE THREAD

**Issue by Imanisima** [Ihuds577/Lab1B#1]
Nice job! Looks good

**Comment by Ihuds577** [comment]
Thanks, so we can just leave an issue thread we don't actually have to make a pull Request?

**Comment by Imanisima** [comment]
Correct! PR is for making changes. You’re not changing anything so leaving a comment under issues is fine

Imani M.

________________________________

**Comment by Ihuds577** [comment]
Yeah I get what its for and all, I was more wondering if the TA and Prof were cool with it. It seems to be the standard right now though. Thanks!
